tbpore/utils.py

```diff
@@ -13,8 +13,8 @@ def is_fastq(p: PathLike) -> bool:
     return Path(p).is_file() and bool(FASTQ_REGEX.search(str(p)))
 
 
-def find_fastq_files(src: Path, recursive: bool = False) -> List[Path]:
-    return [p for p in glob.iglob(f"{src}/**", recursive=recursive) if is_fastq(p)]
+def find_fastq_files(src: Path, recursive: bool = False) -> Set[str]:
+    return set(p for p in glob.iglob(f"{src}/**", recursive=recursive) if is_fastq(p))
```

Comment:
Am I missing something? Where is the import of `glob`? Given `src` is a `Path` might [`rglob`](https://docs.python.org/3/library/pathlib.html#pathlib.Path.rglob) be a more convenient option?